Explorer/Assets/Scripts/CrdtEcsBridge/Serialization/ProtobufSerializer.cs

```diff
@@ -9,5 +9,19 @@ public void DeserializeInto(T instance, in ReadOnlySpan<byte> data)
         {
             instance.MergeFrom(data);
         }
+
+        public ReadOnlyMemory<byte> Serialize(T model)
+        {
+            //Is it the same as doing model.ToByteArray()?
+            /*Span<byte> buffer = new byte[model.CalculateSize()].AsSpan();
+            model.WriteTo(buffer);
+            return buffer;*/
+
+            var buffer = new byte[model.CalculateSize()];
+            var output = new CodedOutputStream(buffer);
```

Comment:
You should write directly to `Span<byte>`, avoid unnecessary allocations such as `CodedOutputStream`